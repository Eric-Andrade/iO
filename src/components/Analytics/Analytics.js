import React, { Component } from 'react'
import styles from './Analytics.css'

class Analytics extends Component {
    // state = {  }
  render () {
    return (
      <div>
        <h2 className={styles.title}>Analytics</h2>
      </div>
    )
  }
}

export default Analytics
