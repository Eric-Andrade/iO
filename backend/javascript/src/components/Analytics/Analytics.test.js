import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Analytics from './Analytics'

test('Analytics component should render as expected', () => {
    const component = shallow(<Analytics />)
    const  tree = toJson(component)
    expect(tree).toMatchSnapshot()
})
