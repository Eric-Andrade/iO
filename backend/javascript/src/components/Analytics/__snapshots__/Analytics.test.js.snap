// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Analytics component should render as expected 1`] = `
<div>
  <h2
    className="title"
  >
    Analytics
  </h2>
</div>
`;
